libs/clients/syslumenn/src/lib/syslumennClient.types.ts

```diff
@@ -177,31 +177,40 @@ export type EstateAsset = {
   description: string
   assetNumber: string
   share: number
+  enabled?: boolean
 }
 
-export type EstateRegistrant = {
+export interface EstateRegistrant extends EstateCommon {
   applicantEmail: string
   applicantPhone: string
-  knowledgeOfOtherWills: 'yes' | 'no'
+  knowledgeOfOtherWills: 'Yes' | 'No'
```

Comment:
I'm curious why this is either 'Yes' or 'No' instead of being a bool, are we expecting more states in the future maybe?

Reply:
Not a frontend guy, but I feel like this was the result of a radio button typing. This can be seen in marriage-conditions and other places but it's a good point and I'll make sure to look into this as we start moving from behind the feature flag. 

Regardless, this will likely be resolved once the data structure and implementation clears up from the initial stages.